client/extensions/zoninator/state/zones/test/actions.js

```diff
@@ -10,11 +10,15 @@ import {
 	ZONINATOR_REQUEST_ZONES,
 	ZONINATOR_REQUEST_ERROR,
 	ZONINATOR_UPDATE_ZONES,
+	ZONINATOR_UPDATE_ZONE,
```

Comment:
You know what I'm gonna say, right? 😉 This will be the last time I'll mention it though.